sessions/js-array-methods/map/index.test.js

```diff
@@ -14,9 +14,9 @@ describe("Mapping Challenge", () => {
   });
   test("questionsAndAnswersTogether", () => {
     expect(questionsAndAnswersTogether).toEqual([
-      '"How often can I use <header>?" - "As often as you like."',
-      '"How often can I use <aside>?" - "As often as you like."',
-      '"On which types can I use destructuring assignment?" - "On Objects and Arrays"',
+      "How often can I use <header>? - As often as you like.",
+      "How often can I use <aside>? - As often as you like.",
+      "On which types can I use destructuring assignment? - On Objects and Arrays",
```

Comment:
👍 
this needs to be changed here as well:
https://github.com/neuefische/web-exercises/blob/2bbd0c7b17a666e925215fb251976cfcbf7a0dad/sessions/js-array-methods/map/index.js#L29

Reply:
True, forgot about that, thank you!